.azure/pipelines/fast-pr-validation.yml

```diff
@@ -1,7 +1,11 @@
+# Don't run CI for this config
+trigger: none
```

Comment:
What's this?

Reply:
This disables the PR validation trigger when we push to branches. PR validation only needs to run on PRs.